django/db/models/sql/compiler.py

```diff
@@ -1383,7 +1391,7 @@ def as_sql(self):
         qn = self.connection.ops.quote_name
         opts = self.query.get_meta()
         insert_statement = self.connection.ops.insert_statement(ignore_conflicts=self.query.ignore_conflicts)
-        result = ['%s %s' % (insert_statement, qn(opts.db_table))]
+        result = ['%s %s' % (insert_statement, self.qualified_table_name(opts.db_table))]
```

Comment:
Have you seen the comment?
https://github.com/django/django/blob/dc9deea8e85641695e489e43ed5d5638134c15c7/django/db/models/sql/compiler.py#L1380-L1382
It looks that you don't need to use `qualified_table_name` here.